tests/behat/show_alerts.feature

```diff
@@ -0,0 +1,45 @@
+@block @block_alerts
+Feature: Show alerts
+  In order to alert users
+  As a user
+  I need to be able to see that the alerts block shows the intended alerts.
+
+  Background:
+    Given the following "blocks" exist:
+      | blockname   | contextlevel | reference  | pagetypepattern | defaultregion |
+      | alerts      | System       | 1          | site-index      | side-pre      |
+    And the following config values are set as admin:
+      | config                  | value                           | plugin        |
+      | title1                  | Alert Title One                 | block_alerts  |
+      | description1            | Alert Description One           | block_alerts  |
+      | link1                   | https://nasa.gov                | block_alerts  |
+      | linktext1               | Alert Link One                  | block_alerts  |
+      | title2                  | Alert Title Two                 | block_alerts  |
```

Comment:
Since there will only be one alert, you can remove the others.

Reply:
done.